From 9e5e7dc3d14f662a3600dea1a766f847a039e9cd Mon Sep 17 00:00:00 2001
From: WRS Support <support@windriver.com>
Date: Fri, 2 Oct 2009 16:12:13 -0400
Subject: [PATCH 04/18] hypervisor: relocate SMP defs for multi users

The enum def needs to be relocated from smp.c into smp.h
so that it can be reused by other places.

Signed-off-by: Liang Li <liang.li@windriver.com>
---
 include/linux/smp.h |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/include/linux/smp.h b/include/linux/smp.h
index cfa2d20..852027c 100644
--- a/include/linux/smp.h
+++ b/include/linux/smp.h
@@ -184,4 +184,9 @@ extern void arch_disable_smp_support(void);
 
 void smp_setup_processor_id(void);
 
+enum {
+	CSD_FLAG_WAIT		= 0x01,
+	CSD_FLAG_ALLOC		= 0x02,
+};
+
 #endif /* __LINUX_SMP_H */
-- 
1.6.5.2

